### PR TITLE
html: rework success rate

### PR DESCRIPTION
**Pull request by apahim**
Success rate to use the following formula:

    ( PASS + WARN ) / ( TOTAL - SKIP - CANCEL ) * 100

Signed-off-by: Amador Pahim <apahim@redhat.com>

### CODE REVIEW

**Comment by apahim** [comment]
[The "pass rate" in html plugin does not include "warn" tests.](https://trello.com/c/6GsdbDLF/1169-the-pass-rate-in-html-plugin-does-not-include-warn-tests)

**Comment by ldoktor** [comment]
Would you please rebase this on the current master so it passes the manual check?

**Comment by apahim** [comment]
Updated

File: optional_plugins/html/avocado_result_html/__init__.py [hunk 1]
```diff
@@ -82,7 +82,8 @@ def warned(self):
         return self.result.warned
 
     def rate(self):
-        total = float(self.result.tests_total)
+        total = float(self.result.tests_total - self.result.skipped -
+                      self.result.cancelled)
```
Comment (lmr) [inline optional_plugins/html/avocado_result_html/__init__.py]: OK, makes sense.

**Pull request merged**